---
arm:
  move_error_pose: [-2.78470, 4.57978, 4.86316, -3.05050, 1.89748, -0.6150]
  # hardcoded pose in front of person
  # Wheelchair
  in_front_person_pose: [-2.30252, 4.23221, 3.84109, -4.65546, 3.94225, 4.26543]
  # Tripod
  # moveIFOPose: [-1.81753, 4.32404, 4.295815, 3.12878, 1.89724, -0.61526]
  # Participant
  # moveIFOPose: [-2.30293, 4.04904, 3.63059, 1.62787, -2.34089, -2.01773]
  # Participant Tripod
  # moveIFOPose: [-1.81752, 4.60286, 4.64300, -3.05122, 1.89743, -0.61493]
  home_config: [-2.11666, 3.34967, 2.04129, -2.30031, -2.34026, 2.9545]
